packages/cli/src/transforms/toktx.ts

```diff
@@ -125,28 +130,29 @@ export const toktx = function (options: ETC1SOptions | UASTCOptions): Transform
 					texture.getURI() ||
 					texture.getName() ||
 					`${textureIndex + 1}/${doc.getRoot().listTextures().length}`;
-				logger.debug(`Texture ${textureLabel} (${slots.join(', ')})`);
+				const prefix = `toktx:texture(${textureLabel})`;
+				logger.debug(`${prefix}: Slots → ${slots.join(', ')}`);
```

Comment:
Looks better with brackets.

```suggestion
				logger.debug(`${prefix}: Slots → [${slots.join(', ')}]`);
```